README_SKINNY.rst

```diff
@@ -2,15 +2,61 @@
 MLflow Skinny: A Lightweight Machine Learning Lifecycle Platform Client
 =======================================================================
 
-MLflow Skinny is a lightweight MLflow package without SQL storage, server, UI, or data science dependencies.
+MLflow Skinny is a lightweight MLflow package without SQL storage, server, UI, or data science dependencies. This is recommended for users who primarily need the tracking and logging capabilities without importing the full suite of MLflow features. 
```

Comment:
@shivp950 @eedeleon  model registry as well? don't want to confuse users into thinking it's experimentation only

Reply:
good point, added it in the update